### DISCUSSION TITLE
Welcome to programming-scala-book-code-examples Discussions!

### DISCUSSION THREAD

**Discussion by deanwampler** [deanwampler/programming-scala-book-code-examples#34] [General]
## 👋 Welcome!

I'm using Discussions as a place to connect with other members of the Scala community, in particular readers of _Programming Scala_. You're also welcome to join the corresponding [Gitter community](https://gitter.im/deanwampler/programming-scala-book-code-examples). Pick your preference ;)

I hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other community members.
  * Welcome others, remain respectful, and are open-minded.

-- Dean

**Comment by hellerim** [comment]
It's september, 2024, and Scal and sbt continue to evolve. Unfortunately, even this book's well-thought examples are not fail-safe w.r.t. to such changes. For newcomers to scala, this is an uncomfortable situation. For instance, at least with Scala 3.5 and sbt 1.10, the command-line behavior was modified which makes some examples of chapter 1 fail (if not more). See [Scala Behavior Changes Make Some Examples Fail](https://github.com/deanwampler/programming-scala-book-code-examples/issues/131).

**Reply by deanwampler** [reply]
Thanks for letting me know and sorry the examples didn't work "out of the box"! I'll add some warning language about "evolving changes" to the README and fix all the examples this coming week.
